### ISSUE TITLE
Allow passing argument to specify release version to "fbs release"

### ISSUE THREAD

**Issue by whattheserver** [mherrmann/fbs#211]
When i have already released on one arch in docker and the version is increased and i want to go into to the next it keeps trying to default increase the version. I think the default action is ok but it should allow passing a defined argument so scripting all the builds can be done

Currently if i try to pass an argument it fails
 ```fbs release 1.0.33
usage: fbs [-h]
           {startproject,run,freeze,sign,installer,sign_installer,repo,upload,release,test,clean,buildvm,runvm,gengpgkey,register,login,init_licensing}
           ...
fbs: error: unrecognized arguments: 1.0.33
```

If there was a way to allow that or a way to say execute fbs freeze + fbs installer + fbs release with the same version for all the docker vms that would be ideal.

So one could accomplish something like this without having to wait for the prompt and manually override the default action of increasing the release a point number
```
fbs runvm ubuntu;
fbs freeze && fbs installer && fbs release 1.0.33 && exit ;
fbs runvm arch;
fbs freeze && fbs installer && fbs release 1.0.33 && exit ;
fbs runvm fedora;
fbs freeze && fbs installer && fbs release 1.0.33 && exit ;
```

**Comment by mherrmann** [comment]
I will happily accept a PR that adds the release number as an optional argument to the `release` command :-)

**Comment by whattheserver** [comment]
So I thought this was going to be easy but it was not letting me set a default value and leave it optional.

I think the best approach is to make it more explicit.

ideally either let it be `fbs release` current and use the current version in the settings.json
`fbs release current`

(venv-fbs) []$ fbs release current
Doing `require 'backports'` is deprecated and will not load any backport in the next major release.
Require just the needed backports instead, or 'backports/latest'.
^C
(venv-fbs) []$

Then make it bump an explicit call vs the default

(venv-fbs) []$ fbs release bump
Release version [1.0.34] : ^C
(venv-fbs) []$

Submitted a PR:
https://github.com/mherrmann/fbs/pull/212

**Comment by mherrmann** [comment]
I now released this in [fbs 0.8.9](https://github.com/mherrmann/fbs/releases/tag/v0.8.9).
